trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative './decorator'
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.length > 10 ? @nameable.correct_name[0..9] : @nameable.correct_na
```

Comment:
- [ ] Please kindly ensure to implement this variable name correctly, it's introducing bug to your app at the moment if I input a name `< 10.`

<img width="778" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/231164510-7f4deadf-b871-42d2-94ec-cba51865972f.png">

```suggestion
    @nameable.correct_name.length > 10 ? @nameable.correct_name[0..9] : @nameable.correct_name
```